plugins/integrations/prometheus/src/main/java/org/apache/cloudstack/metrics/PrometheusExporterImpl.java

```diff
@@ -115,6 +122,10 @@ private void addHostMetrics(final List<Item> metricsList, final long dcId, final
         int total = 0;
         int up = 0;
         int down = 0;
+        Map<String, Integer> total_hosts = new HashMap<>();
```

Comment:
before you touched this method it was already 80 lines, can you please refactor at least the new bits out into smaller chunks?
thanks

Reply:
@DaanHoogland I was thinking about refactoring the whole Prometheus plugin. If I touch one method, I have to change everything. I love to do that, but it would be out of the scope of this pr. I've created #5773 for it.